tests/e2e/instrumentation-dotnet-multicontainer/01-install-app.yaml

```diff
@@ -18,6 +18,6 @@ spec:
     spec:
       containers:
       - name: myapp
-        image: docker.io/adib070/mvc
+        image: docker.io/avadhutp123/aspnetapp:latest
```

Comment:
I do not like an idea to reference private repositories/docked hub repositories here. It should be hosted in scope of OTel/OTel-test dockerhub.

Probably candidate for separate issue.

Reply:
Agree !

Reply:
@pavolloffay  Added the link to the source code repo and have raised the issue for consuming the apps that are hosted on OTEL org. https://github.com/open-telemetry/opentelemetry-operator/issues/1155